examples/README.md

```diff
@@ -6,9 +6,9 @@ This directory contains example apps for each framework we support.
 
 1. Create or Update an example at [`examples/{next,vue,angular}/...`](examples)
 
-   For your `aws-exports.js`, you can [reference an existing or create a new environment](environments).
+   For your `aws-exports.js`, you can [reference an existing or create a new environment](../environments/README.md).
 
-1. Run your example at monorep root: `yarn {next,vue,angular}-example dev`
+1. Run your example at monorep root: `yarn {react,next,vue,angular}-example dev`
```

Comment:
```suggestion
1. Run your example at monorep root: `yarn {react,vue,angular}-example dev`
```

I'd just remove `next`, `react` covers it and is more consistent with the other two.